packages/cozy-konnector-libs/libs/linkBankOperations.spec.js

```diff
@@ -285,4 +285,110 @@ describe('linker', () => {
       })
     })
   })
+
+  describe('linking with combinations', () => {
+    describe('getNotLinkedBills', () => {
+
+      test('it returns the bills that are not linked', () => {
```

Comment:
`it('returns the bills that are not linked')`


`it` can be conveniently into `fit` which will focus jest on only this test.